app/assets/stylesheets/recipe_new.scss

```diff
@@ -42,9 +79,8 @@
   border-radius: 5px;
   box-shadow: 2px 2px 1px darken($light-yellow, 70%);
   font-size: 14pt;
-  margin: 20px 0;
-  padding: 0 5px;
-  padding-top: 15px;
+  margin: 0 0 20px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 20px` instead of `0 0 20px 0`